include/ocpp/helpers/uri.hpp

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 Pionix GmbH and Contributors to EVerest
+#ifndef HELPERS_URI_HPP
+#define HELPERS_URI_HPP
+
+#include <string>
+
+namespace helpers {
+
+    std::string URIAppendPath(std::string uri, std::string path);
```

Comment:
we decided to use snake_case for everything we add. Libocpp needs a general refactor to make this consistent